assists/baremetalconfig_xlnx.py

```diff
@@ -203,24 +205,55 @@ def is_compat(node, compat_string_to_test):
         return xlnx_generate_bm_config
     return ""
 
+def get_stdin(sdt, chosen_node, node_list):
+    prop_dict = Lopper.node_properties_as_dict(sdt.FDT, chosen_node.abs_path)
```

Comment:
Is the chosen node not being properly pulled into the SDT already ? If not, that's an issue that I need to address.

We shouldn't need to go to the FDT at all to process parts of the tree.

Reply:
I have fixed it and force pushed the change. Please take a look into it.